Tiltfile

```diff
@@ -165,7 +165,7 @@ def load_provider_tiltfiles():
 
 tilt_helper_dockerfile_header = """
 # Tilt image
-FROM golang:1.20.10 as tilt-helper
+FROM golang:1.20.11 as tilt-helper
```

Comment:
Can we update the version on L178 as well?

Reply:
Good catch @killianmuldoon , my fault assuming we have 1.20.10 everywhere when searching for it in IDE 🤦🏼‍♂️

Reply:
Yeah - I think I did the exact same for the last bump or even more :sweat_smile: